style.css

```diff
@@ -394,33 +368,76 @@ span {
   margin-bottom: 20px;
 }
 
-.input-textarea {
-  display: flex;
-  flex-direction: row;
-  width: 100%;
-  height: 150px;
+textarea {
+  width: 447px;
+  height: 170px;
   background: #fff;
   border: 1px solid #cfd8dc;
   border-radius: 8px;
   font-family: “Poppins”, sans-serif;
   font-style: normal;
   font-weight: 400;
-  font-size: 15px;
-  line-height: 24px;
-  color: #b3bac5;
+  font-size: 17px;
+  line-height: 20px;
+  color: #172b4d;
   padding: 15px 38px 15px 16px;
+  margin-bottom: 20px;
+}
+
+.intro {
+  width: 100%;
+  height: 56px;
+  font-family: “Poppins”, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  color: #fff;
+  align-items: center;
+  justify-content: center;
 }
 
-.submit-button {
-  width: 140px;
-  height: 48px;
+.intro-contact {
+  font-family: “Poppins”, sans-serif;
+  font-style: normal;
+  text-align: center;
+  color: #ebebff;
+  justify-content: center;
+}
+
+#Contact {
+  display: flex;
+  align-content: center;
+  align-items: center;
+  flex-wrap: wrap;
+}
+
+#button {
+  width: 150px;
+  height: 40px;
+  font-family: “Poppins”, sans-serif;
+  font-style: normal;
   font-weight: 500;
+  font-size: 25px;
   line-height: 24px;
-  border-radius: 8px;
-  background: #fff;
-  margin-bottom: 30px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  transition-duration: 0.4s;
```

Comment:
- [ ] Kindly adhere to the hover color pattern given , as your current reactions does not tally with the template olors below :

## The hover colors:
![image](https://user-images.githubusercontent.com/50990925/220592723-8772b2a3-bcf3-4d56-b7f7-e6a7c16d89ad.png)